librepo/downloader.h

```diff
@@ -100,6 +100,7 @@ lr_download_single_cb(GSList *targets,
                       gboolean failfast,
                       LrProgressCb cb,
                       LrMirrorFailureCb mfcb,
+                      LrStartTransferCb stcb,
```

Comment:
It looks like API. And I think this is not good to break it if there still could be a compatible alternative.

Reply:
How would you achieve a compatible alternative?
The possible solution (maybe) I can think about is to call progresscb with some special values, which upper layers can decode / recognize. But I think it would be an obfuscated solution, which contains a non obvious "secrets". IMO it is cleaner to bump a version.